### DISCUSSION TITLE
Cygwin DLL error

### DISCUSSION THREAD

**Discussion by areteeem** [community/community#83075] [New to GitHub]
Hi there, so I recently started working on my project on unity and git stuck with my team on a one big issue. **So I can't clone other people repositories, even if i am an collaborator**. I tried reinstalling github desktop, but it didn't help. It always shows cygwin dll error. It says "his problem is probably due to using incompatible versions of the cygwin DLL." _I'll appreciate any help!!_

**Comment by ShutovKS** [comment]
Check Git Installation:
Make sure that you have Git installed on your machine and that it's up-to-date. You can download the latest version of Git from the official Git website.

**Reply by areteeem** [reply]
I did check it's existence

**Reply by ShutovKS** [reply]
@areteeem Have you tried removing it and installing it again?

**Reply by areteeem** [reply]
Yep. The error has been already solved, by re-installing github desktop to another directory.

**Comment by Karan60** [comment]
Please Response #1

**Comment by Karan60** [comment]
Please Response #2
